### DISCUSSION TITLE
제페토스크립트 예제가 실행이 안됩니다. (EventTriggerSample)

### DISCUSSION THREAD

**Discussion by mayko1234** [naverz/zepeto-studio-kor#228] [5. Q&A]
https://studio.zepeto.me/guides/eventtrigger

>         trigger.triggers.Add(entry);

이 라인에서 Property 'Add' does not exist on type 'List$1<any>라는 에러가 발생합니다. 도와주세요~!

**Comment by moondory77-naverz** [comment]
안녕하세요.
다음 샘플프로젝트의 7.EventTrigger 항목을 참고 부탁드리겠습니다.
[https://github.com/naverz/zepeto-script-sample](https://github.com/naverz/zepeto-script-sample)
(가능한 ZEPETO.World 패키지는, 최신으로 업데이트 후 확인 부탁드리겠습니다. - 현 기준 1.1.2)

도움이 될 수 있다면 좋겠습니다.
감사합니다.

**Reply by mayko1234** [reply]
감사합니다~!

**Comment by hyeonjin-jo** [comment]
안녕하세요.

아래의 답변 참고 부탁드립니다.

https://github.com/naverz/zepeto-studio-kor/discussions/164

**Reply by mayko1234** [reply]
감사합니다! index.d.ts파일 추가 후 에러없이 플레이됩니다. 🙌👍
